common-lib/src/cmr/common/dev/util.clj

```diff
@@ -37,11 +37,10 @@
     (run! #(-> % str touch-file) files)))
 
 (defn speak
-  "Says the specified text outloud."
-  ([text]
-   (speak "Victoria" text))
-  ([voice text]
-   (sh/sh "say" "-v" voice text)))
+  "Says the specified text outloud. This can be used for testing threads which may not print
```

Comment:
I did not write this, I found it in CMR it's very old, but I got it to work and I used it a lot. It's for real, give it a try!

Reply:
That's really interesting! Looks like the 'say' command might just be mac

Reply:
yes, it is Mac only. I actual had to use this to debug a multi thread process that lost access to standard out